<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculadoraConversion</class>
 <widget class="QMainWindow" name="CalculadoraConversion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Caluladora de conversion</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lEuros">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Euros:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lLempiras">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Lempiras:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eEuros">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eLempiras">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lRatios">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ratios:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lRatioEurHnl">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0 EUR/HNL</string>
    </property>
   </widget>
   <widget class="QLabel" name="lRatioHnlEur">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0 HNL/EUR</string>
    </property>
   </widget>
   <widget class="QLabel" name="lUltimaActualizacion">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ultima actualizacion:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lFechaUltimaActualizacion">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>9/9/2019</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cTipoCambio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Euros -&gt; Lempiras</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lempiras -&gt; Euros</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="bConvertir">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Convertir</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
